src/containers/pages/InterventionPlan/NewPlan/General/tests/index.test.tsx

```diff
@@ -29,8 +29,7 @@ describe('containers/pages/NewPlan', () => {
     expect(toJson(wrapper.find('h3.mb-3.page-title'))).toMatchSnapshot('page title');
 
     // check that PlanForm receives the correct props
-    expect(wrapper.find('PlanForm').props()).toEqual({
-      ...planFormProps,
+    expect(wrapper.find('PlanForm').props()).toMatchSnapshot({
```

Comment:
not sure if this is a controversial move -- seemed like the task here was to update the fixtures so moved to snapshot, perhaps there is more to pull out and put in the expect map here if we want to be more deterministic (or not) for parts of this content

Reply:
We've been trying to prefer using fixtures over large snapshots.  It's historically been harder to deal with snapshots when they are large i.e. engineers are tempted to just update the snapshot instead of trying to figure out why thousands of lines of the snapshot have a diff.  More ideas on this here: https://kentcdodds.com/blog/effective-snapshot-testing/

Reply:
I changed this back to using a much smaller fixture (turns out we could import it from elsewhere)